src/amp/components/Analytics.test.tsx

```diff
@@ -2,33 +2,34 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { Analytics, AnalyticsModel } from '@root/src/amp/components/Analytics';
 
-describe('AnalyticsComponent', () => {
-	const analytics: AnalyticsModel = {
-		gaTracker: 'UA-XXXXXXX-X',
-		title: 'Foo',
-		fbPixelaccount: 'XXXXXXXXXX',
-		comscoreID: 'XXXXXXX',
-		contentType: 'Article',
-		id: 'some/page',
-		beacon: `localhost/count/pv.gif`,
-		neilsenAPIID: 'XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXX',
-		domain: 'amp.theguardian.com',
-		permutive: {
-			namespace: 'guardian',
-			apiKey: '42-2020',
-			payload: {
-				'properties.content.title': 'article title',
-			},
+const analyticsBase: AnalyticsModel = {
+	gaTracker: 'UA-XXXXXXX-X',
+	title: 'Foo',
+	fbPixelaccount: 'XXXXXXXXXX',
+	comscoreID: 'XXXXXXX',
+	contentType: 'Article',
+	id: 'some/page',
+	beacon: `localhost/count/pv.gif`,
+	neilsenAPIID: 'XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXX',
+	domain: 'amp.theguardian.com',
+	permutive: {
+		namespace: 'guardian',
+		apiKey: '42-2020',
+		payload: {
+			'properties.content.title': 'article title',
 		},
-		ipsosSectionName: 'section',
-	};
-
-	beforeEach(() => {
-		delete analytics.section;
-	});
```

Comment:
removed this pattern in favor of spreading `analyticsBase` and changing `section` when needed